### ISSUE TITLE
Update to latest version within major or minor

### ISSUE THREAD

**Issue by dimafeng** [tjunnone/npm-check-updates#252]
In my project, I use direct dependencies and occasionally I need to update minor versions of my dependencies.
When I use `ncu -u` I get: 
```
async                  1.5.0  →  2.0.0-rc.6
gulp-jshint            1.9.0  →       2.0.1 
```
It would be convenient to have some flags which can help to achieve following behavior:
```
$ ncu -u --major
async                  1.5.0  →  1.6.0
```
and
```
$ ncu -u --minor
async                  1.5.0  →  1.5.1
```

**Comment by raineorshine** [comment]
Hi! Thanks for the suggestion. I'd be open to that change. If you or anyone feels an interested in working on that feature, let me know!

**Comment by dimafeng** [comment]
I'm working on it

**Comment by raineorshine** [comment]
Published in v2.7.0. Thanks!
